client/rest-high-level/src/test/java/org/elasticsearch/client/IndicesRequestConvertersTests.java

```diff
@@ -868,6 +868,12 @@ private void resizeTest(ResizeType resizeType, CheckedFunction<ResizeRequest, Re
         ResizeRequest resizeRequest = new ResizeRequest(indices[0], indices[1]);
         resizeRequest.setResizeType(resizeType);
         Map<String, String> expectedParams = new HashMap<>();
+
+        if (randomBoolean()) {
+            resizeRequest.setCopySettings(Boolean.TRUE);
```

Comment:
```suggestion
            resizeRequest.setCopySettings(true);
```

Reply:
Fixed in https://github.com/elastic/elasticsearch/pull/39752/commits/b20c245d526abdad6902a2d467b1e1ef9c30ab76